JhipsterBlazor/Program.cs

```diff
@@ -49,6 +50,14 @@ public static async Task Main(string[] args)
             builder.Services.AddSingleton<IRegionService, RegionService>();
             builder.Services.AddSingleton<IRegisterService, RegisterService>();
 
+            /* #### Http Interceptor #####*/
+            builder.Services.AddHttpClientInterceptor();
+            builder.Services.AddTransient(sp => new HttpClient
+            {
+                BaseAddress = new Uri(builder.HostEnvironment.BaseAddress)
+            }.EnableIntercept(sp));
```

Comment:
I think this is incompatiblewith the authentication
```
 private async Task SetUserAndAuthorizationHeader(JwtToken jwtToken)
        {
            IsAuthenticated = true;
            _httpClient.DefaultRequestHeaders.Remove(AuthorizationHeader);
            _httpClient.DefaultRequestHeaders.Add(AuthorizationHeader, $"Bearer {jwtToken.IdToken}");
            try
            {
                CurrentUser = await _httpClient.GetFromJsonAsync<UserModel>(AccountUrl);
            }
            catch
            {
                IsAuthenticated = false;
            }
            NotifyAuthenticationStateChanged(GetAuthenticationStateAsync());
        }
```
I use singleton to set all request headers when the user was authenticated